Labs/Lab2/README.md

```diff
@@ -22,7 +24,7 @@ Deployment is an abstractions on pods and replica set.
 
 ## Display information about a specific deployment
 
-`kubectl describe deployment guestbook-deployment`
+`kubectl describe deployment guestbook-v1`
```

Comment:
Thanks Ed!! When I was testing, I meant to remove v1 from the .yaml file but I forgot to update. Let's keep it with v1.